pyaedt/edb_grpc/core/components.py

```diff
@@ -0,0 +1,2209 @@
+            self.add_port_on_rlc_component(component.refdes)
+            return True
+        else:
+            return self.set_component_rlc(component.refdes)
+
+    @pyaedt_function_handler()
+    def add_port_on_rlc_component(self, component=None):
+        """Deactivate RLC component and replace it with a circuit port.
+        The circuit port supports only 2-pin components.
+
+        Parameters
+        ----------
+        component : str
+            Reference designator of the RLC component.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
+        if isinstance(component, str):  # pragma: no cover
+            component = self.components[component]
+        if not isinstance(component, EDBComponent):  # pragma: no cover
+            return False
+        self.set_component_rlc(component.refdes)
+        pins = self.get_pin_from_component(component.refdes)
+        if len(pins) == 2:  # pragma: no cover
+            pos_pin_loc = self.get_pin_position(pins[0])
+            pt = self._pedb.edb.Geometry.PointData(
+                self._get_edb_value(pos_pin_loc[0]), self._get_edb_value(pos_pin_loc[1])
+            )
+            pin_layers = self._padstack._get_pin_layer_range(pins[0])
+            pos_pin_term = self._pedb.edb.Cell.Terminal.PointTerminal.Create(
+                self._active_layout, pins[0].GetNet(), pins[0].GetName(), pt, pin_layers[0]
+            )
+            if not pos_pin_term:  # pragma: no cover
+                return False
+            neg_pin_loc = self.get_pin_position(pins[1])
+            pt = self._pedb.edb.Geometry.PointData(
+                self._get_edb_value(neg_pin_loc[0]), self._get_edb_value(neg_pin_loc[1])
+            )
+            neg_pin_term = self._pedb.edb.Cell.Terminal.PointTerminal.Create(
+                self._active_layout, pins[1].GetNet(), pins[1].GetName() + "_ref", pt, pin_layers[0]
+            )
+            if not neg_pin_term:  # pragma: no cover
+                return False
+            pos_pin_term.SetBoundaryType(self._pedb.edb.Cell.Terminal.BoundaryType.PortBoundary)
+            pos_pin_term.SetIsCircuitPort(True)
+            pos_pin_term.SetName(component.refdes)
+            neg_pin_term.SetBoundaryType(self._pedb.edb.Cell.Terminal.BoundaryType.PortBoundary)
+            neg_pin_term.SetIsCircuitPort(True)
+            pos_pin_term.SetReferenceTerminal(neg_pin_term)
+            self._logger.info("Component {} has been replaced by port".format(component.refdes))
+            return True
+
+    @pyaedt_function_handler()
+    def _create_pin_group_terminal(self, pingroup, component=None, isref=False):
+        """Creates an EDB pin group terminal from a given EDB pin group.
+
+        Parameters
+        ----------
+        pingroup : Edb pin group.
+
+        component : str or EdbComponent
+
+        isref : bool
+
+        Returns
+        -------
+        Edb pin group terminal.
```

Comment:
```suggestion
        pingroup : EDB pin group.

        component : EDB component object or EDB component name.

        isref : bool

        Returns
        -------
        EDB pin group terminal.
```
Complete descriptions are needed.